Data/IntSet.hs

```diff
@@ -89,16 +93,17 @@ module Data.IntSet (
             , isProperSubsetOf
             , disjoint
 
-            -- * Construction
-            , empty
-            , singleton
+            -- ** Insertion
             , insert
```

Comment:
Insertion and deletion should go right after construction.

Reply:
Done.